packages/components/src/Hero/constants.ts

```diff
@@ -0,0 +1,7 @@
+export enum homepageHeroText {
```

Comment:
FYI, I think we're going to go with a different way of rendering these text strings. I'm working on it in a PR now, but we can update these strings in the future.